demisto_sdk/commands/common/content/objects/pack_objects/modeling_rule/modeling_rule.py

```diff
@@ -0,0 +1,28 @@
+from typing import Union
+
+import demisto_client
+from wcmatch.pathlib import Path
+
+from demisto_sdk.commands.common.constants import (MODELING_RULE, FileType)
+from demisto_sdk.commands.common.content.objects.pack_objects.abstract_pack_objects.yaml_unify_content_object import \
+    YAMLContentUnifiedObject
+
+
+class ModelingRule(YAMLContentUnifiedObject):
+    def __init__(self, path: Union[Path, str]):
+        super().__init__(path, FileType.MODELING_RULE, MODELING_RULE)
+
+    def upload(self, client: demisto_client):
+        """
+        Upload the modeling_rule to demisto_client
+        Args:
+            client: The demisto_client object of the desired XSOAR machine to upload to.
+
+        Returns:
+            The result of the upload command from demisto_client
+        """
+        # return client.import_modeling_rules(file=self.path)
+        pass
```

Comment:
make sure not to forget these

Reply:
We do not currently support uploading new XSIAM entities, we will do (https://github.com/demisto/etc/issues/47165)